src/auth/auth-manager.ts

```diff
@@ -47,10 +47,11 @@ export const AuthManager = () => {
   };
 
   const isValidIdToken: () => boolean = () => {
-    const { idTokenClaims }: any = msalInst?.getActiveAccount() || null;
+    const accessToken: string = getAccessToken() || '';
```

Comment:
You have changed the names of the variables to accessToken but still the method name is `isValidIdToken`. We have to align the name of the variables and the method itself.